### PR TITLE
fixing bug artifacts from XDR missing

### PR DESCRIPTION
**Pull request by maimorag**
## Status
- [x] In Progress

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-36647)

## Description
- Fixing incident artifacts is not populated on new incidents when bringing them into XSOAR.
- Setting exclude_fields with default_value=False

### CODE REVIEW

**Review by eyalpalo — CHANGES_REQUESTED** [review]

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py [hunk 1]
```diff
@@ -569,7 +569,7 @@ def get_incident_extra_data_command(client, args):
 
         else:  # the incident was not modified
             return "The incident was not modified in XDR since the last mirror in.", {}, {}
-    raw_incident = client.get_multiple_incidents_extra_data(incident_id_list=[incident_id])
+    raw_incident = client.get_multiple_incidents_extra_data(incident_id_list=[incident_id], fields_to_exclude=False)
```
[the remark this answers is not in the document]
Comment (maimorag) [reply Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:572]: ok, add a parameter for the function if the customer rather return artifacts or not

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py [hunk 2]
```diff
@@ -1224,10 +1225,15 @@ def main():  # pragma: no cover
...
-                                                  demisto.getLastRun().get('next_run'), max_fetch, statuses, starred,
-                                                  starred_incidents_fetch_window,
-                                                  fields_to_exclude)
+            next_run, incidents = fetch_incidents(client=client,
+                                                  first_fetch_time=first_fetch_time,
+                                                  integration_instance=integration_instance,
+                                                  exclude_artifacts=exclude_artifacts,
+                                                  last_run=demisto.getLastRun().get('next_run'),
+                                                  max_fetch=max_fetch,
+                                                  statuses=statuses, starred=starred,
```
Comment (eyalpalo) [inline Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py:1234]:
```suggestion
                                                  statuses=statuses,
                                                  starred=starred,
```

**Review by eyalpalo — APPROVED** [review]

**Comment by maimorag** [comment]
test playbook "test Playbook - Cortex XDR - Retrieve File by sha256 " fails on 
https://jira-dc.paloaltonetworks.com/browse/CIAC-9932.

**Comment by maimorag** [comment]
test playbook "test Playbook - Cortex XDR - Retrieve File by sha256 " 
 fails on each [server](https://gitlab.xdr.pan.local/xdr/cortex-content/content/-/jobs/6147595) on:
 https://jira-dc.paloaltonetworks.com/browse/CIAC-9932.

@ShahafBenYakir can you please FM?

**Pull request merged**